pinot-core/src/test/java/org/apache/pinot/core/realtime/impl/RealtimeSegmentStatsHistoryTest.java

```diff
@@ -222,9 +222,9 @@ public void testMultiThreadedUse() throws Exception {
   // This test attempts to ensure that future modifications to RealtimeSegmentStatsHistory does not prevent the software
   // from reading data serialized by earlier versions. The serialized data has one segment, with 2 columns -- "v1col1" and
   // "v1col2".
-  //@Test
+  @Test
   public void testVersion1() throws Exception {
-    final String fileName = "realtime-segment-stats-history-v1.ser";
+    final String fileName = "realtime-segment-stats-history.ser";
```

Comment:
I would retain the old name, because the _contents_ of this file is the actual V1 contents,